packages/frontend/src/pages/Playground.spec.ts

```diff
@@ -314,7 +314,7 @@ test('sending prompt should display the prompt and the response', async () => {
         {
           role: 'assistant',
           id: 'message-2',
-          content: 'a response from the assistant',
+          content: 'a response from the assistant\neat, sleep, code, repeat\neat, sleep, code, repeat',
```

Comment:
Thanks for the regression test 👍

Reply:
np :wink: 

I was wondering to create a separate test, but wanted to get the fix in ASAP. Might come up in a future PR :shrug: